app/views/foreman_puppet_enc/hosts/puppet.erb

```diff
@@ -0,0 +1 @@
+Welcome to <b>ForemanPuppetEnc</b>
```

Comment:
Hmm :thinking: sound like warming welcome, but not sure where we are using it :smiley_cat:

Reply:
@ezr-ondrej I removed this file :) thanks